client/src/Movies/MovieList.js

```diff
@@ -1,5 +1,7 @@
-import React, { Component } from 'react';
-import axios from 'axios';
+import React, { Component } from "react";
+import axios from "axios";
+
+import { Link } from 'react-router-dom';
 
 export default class MovieList extends Component {
   constructor(props) {
```

Comment:
Here is where you would start to manage state on this component if you were to complete stretch.